queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -72,6 +73,7 @@
 APP_CONFIG = config.get_named_config(os.getenv('DEPLOYMENT_ENV', 'production'))
 FLASK_APP = Flask(__name__)
 FLASK_APP.config.from_object(APP_CONFIG)
+setup_versioning()
```

Comment:
As per suggested, call to enable versioning switching before data model initialized